Sources/ObjectivelyMVC/Slider.c

```diff
@@ -202,6 +206,8 @@ static Slider *initWithFrame(Slider *self, const SDL_Rect *frame, ControlStyle s
 		self->label->view.alignment = ViewAlignmentMiddleRight;
 		self->label->view.padding.left = DEFAULT_SLIDER_LABEL_PADDING;
 
+		self->snapToStep = DEFAULT_SLIDER_SNAP_TO_STEP;
```

Comment:
No need to even include this. All instances in Objectively are initialized with `0`, so for things that default to false, I don't even mention them in the initializers.